component/file_cache/file_cache_test.go

```diff
@@ -991,15 +965,9 @@ func (suite *fileCacheTestSuite) TestOpenCloseHandleCount() {
 
 func (suite *fileCacheTestSuite) TestOpenPreventsEviction() {
 	defer suite.cleanupTest()
-	// Setup
-	suite.cleanupTest() // teardown the default file cache generated
```

Comment:
Why are cleanupTest and setupTestHelper deleted?

Reply:
I changed the default config to have a 1s timeout, so now this test can use the default.